app/Classes/Facades/StringHelper.php

```diff
@@ -9,6 +9,8 @@ public static function toReadableStatus($value){
         $data = [
             'pending_building_inspection' => 'Pending for Building Inspection',
             'pending_waterworks_inspection'=>'Pending for Waterworks  Inspection',
+            'pending_for_payment'=>'Pending for Payments',
+            'pending' => 'Pending',
```

Comment:
We don't use the StringHelper class anymore. Please check the Service model for the status